Library/Formula/gtkglext.rb

```diff
@@ -14,126 +13,133 @@ class Gtkglext < Formula
     sha1 "03c196737dbef40fef3bd51f7feba9040a039d3a" => :mountain_lion
   end
 
-  depends_on 'pkg-config' => :build
-  depends_on 'glib'
-  depends_on 'gtk+'
-  depends_on 'pangox-compat'
-  depends_on :x11
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "gtk+"
+  depends_on "pangox-compat"
 
-  # fixes an incompatibility with recent GTK versions
-  # patch from: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=585155
-  patch :DATA
+  patch :p0 do
+    url "https://trac.macports.org/export/132848/trunk/dports/devel/gtkglext/files/patch-configure.diff"
+    sha256 "aca35cd6ae28613b375301068715f82b59bd066a32b2f4d046177478950ab026"
+  end
 
-  def install
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make install"
+  patch :p0 do
+    url "https://trac.macports.org/export/132848/trunk/dports/devel/gtkglext/files/patch-examples-pixmap-mixed.c.diff"
+    sha256 "d2fe00bfcf96b3c78dd4b01aa119a7860a34ca6080c57f0ccc7a8e2fc4a3c92b"
+  end
+
+  patch :p0 do
+    url "https://trac.macports.org/export/132848/trunk/dports/devel/gtkglext/files/patch-examples-pixmap.c.diff"
+    sha256 "d955b18784d3e83c1f698e63875d98de5bad9eae1e84b66549dfe25d9ff94d51"
+  end
+
+  patch :p0 do
+    url "https://trac.macports.org/export/132848/trunk/dports/devel/gtkglext/files/patch-gdk-gdkglglext.h.diff"
+    sha256 "a1b6a97016013d5cda73760bbf2a970bae318153c2810291b81bd49ed67de80b"
+  end
+
+  patch :p0 do
+    url "https://trac.macports.org/export/132848/trunk/dports/devel/gtkglext/files/patch-gdk-gdkglquery.c.diff"
+    sha256 "a419b8d523f123d1ab59e4de1105cdfc72bf5a450db8031809dcbc84932b539f"
+  end
+
+  patch :p0 do
+    url "https://trac.macports.org/export/132848/trunk/dports/devel/gtkglext/files/patch-gdk-gdkglshapes.c.diff"
+    sha256 "bc01fccec833f7ede39ee06ecc2a2ad5d2b30cf703dc66d2a40a912104c6e1f5"
```

Comment:
Can you please submit these patches to the upstream developers of this project and add links to the upstream patch submissions and explanation of why the patches are needed in a comment in the formula file. Thanks!